packages/trader/src/AppV2/Components/OnboardingGuide/GuideForPages/onboarding-video.tsx

```diff
@@ -21,7 +22,7 @@ const OnboardingVideo = ({ type }: TOnboardingVideoProps) => {
 
     return (
         <div className={clsx('guide__player__wrapper', is_loading && 'guide__player__wrapper--is-loading')}>
-            {is_loading && <Skeleton.Square height={218.5} />}
+            {is_loading && <Skeleton.Square height={`calc(100vw * ${ASPECT_RATIO})`} />}
```

Comment:
Because the component was build the way, that you need to pass height as prop. If I add a className and do the same, but in css file, it will be overwritten by default height value (100%) which is applied inline. I don't want to add !important and decided to pass height as it is.
<img width="559" alt="Screenshot 2024-10-03 at 9 29 53 AM" src="https://github.com/user-attachments/assets/413ac722-c82b-4faa-a32b-3bb73cc46b3c">